src/main/java/seedu/address/logic/parser/FindEventsCommandParser.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+
+import seedu.address.logic.commands.FindEventsCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.event.EventTitleContainsKeywordsPredicate;
+
+/**
+ * Parses input arguments and creates a new FindEventsCommand object
```

Comment:
Just a missing fullstop
```suggestion
 * Parses input arguments and creates a new FindEventsCommand object.
```